contrib/docker/entrypoint.sh

```diff
@@ -55,4 +55,4 @@ fi
 zcash-fetch-params
 touch .zcash/zcash.conf
 echo "Starting: ${ZCASHD_CMD}"
-eval exec "${ZCASHD_CMD}" "${@}"
+exec "${ZCASHD_CMD}" "${@}"
```

Comment:
This change is wrong.

Reply:
I think `eval "exec ${ZCASHD_CMD} ${*}"` will do the right thing. Going to test it, but if you know or have a recommendation …

Reply:
For now let's just add an annotation to allow the `eval` here, and then open an issue to fix the bug.

Reply:
https://github.com/zcash/zcash/issues/6297